### PR TITLE
Editor/sounds : Add sound playback & visualization

### PR DESCRIPTION
**Pull request by Vroonsh**
image
 - Added UnityEngine.AudioModule.dll to Scripts\.unity and update references for UnityEditor.vcsproj
 - Added SoundExtensions to VPE.Unity to convert SoundData to [-1;1] floating values Audio data used by AudioClip
 - Added Sound data details & visualization window 
 - Added Play button and playback management through AudioSource object created while SoundManager is present.

### CODE REVIEW

File: VisualPinball.Unity/VisualPinball.Unity.Editor/Managers/SoundManager.cs [hunk 1]
```diff
@@ -109,15 +111,59 @@ protected override void OnEnable()
...
 		}
 
+		public static string[] _soundOutTypeStrings = {
+			"Table",
+			"Backglass",
+		};
+		private static byte[] _soundOutTypeValues = {
+			SoundOutTypes.Table,
+			SoundOutTypes.Backglass,
+		};
```
Comment (jsm174) [inline VisualPinball.Unity/VisualPinball.Unity.Editor/Managers/SoundManager.cs:134]: Maybe move these more to the top of the file? And would `_soundOutTypeStrings` be private too?

**Review by jsm174 — COMMENTED** [review]
Nothing major, just a few minor suggestions. Looks good!

**Comment by Vroonsh** [comment]
Found a little issue where the AudioSource temporary object was saved in the Scene, made it also hidden from hierarchy to avoid misunderstanding.

**Review by jsm174 — APPROVED** [review]
Awesome!

**Review by freezy — APPROVED** [review]
Thanks!

**Pull request merged**